packages/design-system-react/src/utils/tw-merge.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable tailwindcss/no-contradicting-classname */
```

Comment:
We are actively testing how twMerge utility handles contradictory classnames so we want to disable the eslint rule for this test file